pantheon-bundle/frontend/src/app/__snapshots__/searchBeta.test.tsx.snap

```diff
@@ -56,6 +58,7 @@ exports[`SearchBeta tests should render default Search component 1`] = `
           <InputGroup>
             <SearchInput
               aria-label="search input"
+              className="filters-toolbar__name-search"
```

Comment:
I would think this one would be `filters_bar` instead of toolbar? Is it part of the same parent component as `filters-bar__filters-wrapper`?